### PR TITLE
ICS calendar event export

### PR DESCRIPTION
**Pull request by benjaoming**
Had to learn a lot of ruby and rails stuff to do this :)

I've tested that the times are showing correctly in Thunderbird.

### CODE REVIEW

File: app/views/events/show.ics.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+X-WR-CALNAME:<%= t('events.ics.prodid') %>
+X-WR-TIMEZONE:"UTC"
+BEGIN:VEVENT
+DTSTAMP:<%= DateTime.now.strftime('%Y%m%dT%H%M%SZ') %>
+UID:<%= @event.id %>
+DTSTART:<%= @event.start_time.utc.strftime('%Y%m%dT%H%M%SZ') %>
+DTEND:<%= @event.end_time.utc.strftime('%Y%m%dT%H%M%SZ') %>
+SUMMARY:<%= @event.title %>
+DESCRIPTION:<%= @event.short_description %>
+<%= @event.long_description %>
```
Comment (benjaoming) [inline app/views/events/show.ics.erb]: If we can somehow strip html tags here, that'd be nice.. on the other hand, are there any html tags in these contents??

Comment (ronan-mch) [inline app/views/events/show.ics.erb]: Shouldn't be.

**Comment by ronan-mch** [comment]
:horse_racing: Hold your horses there partner, looks like you've based this off master rather than beta... I'd rather get it into beta so we don't need to worry about managing active development in two branches at once. I'll have a look at cherry-picking these commits into beta, rather than merging this branch into master as you suggest.

**Comment by benjaoming** [comment]
@ronan-mch that's because beta wasn't working so I couldn't develop a new feature for it, it was totally intentional!

**Pull request closed without merging**